examples/misc/lib/effective_dart/design_good.dart

```diff
@@ -3,7 +3,6 @@
 import 'dart:async';
```

Comment:
Remove the `import 'dart:async'`?

Reply:
Good idea! I just tried and it won't work, though — both files use `FutureOr`.